lib/fog/ovirt/compute.rb

```diff
@@ -50,73 +50,53 @@ def respond_to_missing?(method_name, include_private = false)
         end
       end
 
-      require "fog/ovirt/compute/v3"
       require "fog/ovirt/compute/v4"
 
       def self.new(options = {})
         super(options)
-
-        # rubocop:disable Style/ConditionalAssignment
-        if options[:api_version] == "v4"
-          @client = Fog::Ovirt::Compute::V4.new(options)
-        else
-          @client = Fog::Ovirt::Compute::V3.new(options)
-        end
-        # rubocop:enable Style/ConditionalAssignment
+        @client = Fog::Ovirt::Compute::V4.new(options)
       end
 
-      # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
       def method_missing(symbol, *args)
         @client.__send__(symbol, *args)
+        super
```

Comment:
Why do you want to call `Object.method_missing` after calling the same method on `@client`?